### PR TITLE
add login and logout functionality to navbar

### PR DESCRIPTION
**Pull request by seann1**
added links to sign in and sign out auth api endpoints to the navbar. Github sign in and out appears to be working from the navbar.

### CODE REVIEW

File: frontend/src/Navbar.tsx [hunk 1]
```diff
@@ -88,25 +88,22 @@ const Navbar: FunctionComponent = () => {
                 onClick={() => setProfileDiv(!profileDiv)}
               />
               {profileDiv && (
-                <div className="absolute right-0 mt-2 w-48 bg-white rounded-md overflow-hidden shadow-xl z-20 mr-2">
-                  <span className="block px-4 py-2 text-sm text-gray-800 border-b hover:bg-gray-200">
+                <div className="absolute right-0 mt-2 w-48 bg-white rounded-md overflow-hidden shadow-xl z-20 mr-2 cursor-pointer">
+                  <a
```
Comment (mtso) [inline frontend/src/Navbar.tsx:92]: i wonder if we can use the `reloadDocument` property in Link to simulate <a href> 

https://reactrouter.com/docs/en/v6/api#link

**Review by mtso — APPROVED** [review]
Looks good! I left a comment but i think it is okay to approve as-is :)

**Comment by seann1** [comment]
I'll add reloaddocument and look into logging out via fetch instead of an `<a href>`

**Review by mtso — APPROVED** [review]
new changes LGTM!!

**Pull request merged**